Core/Dto/CharacterSeparatedString.cs

```diff
@@ -31,5 +31,10 @@ public override string ToString()
                 .Where(item => !string.IsNullOrEmpty(item))
                 .ToList();
         }
+
+        public char GetSeparatorIn()
```

Comment:
If you use the implicit converter to convert this type to a List<string>, this method won't be needed anymore.

Reply:
Done